scripts/arangodb-docker-container-setup/docker-container-scripts/test-data/testData.json

```diff
@@ -557,7 +557,7 @@
                     "allowedUserIds": [],
                     "allowedGroupIds": []
                 },
-                "audioFilename": "3948.mp3",
+                "audioFilename": "Kwa",
```

Comment:
Kwa.mp3 or Kwa.wav maybe? Fix all...

Reply:
If you look at `buildPhotographTestData.ts`, the filename has no extension.  The extension is added in `photograph.view-model.ts` on line 34: `this.imageURL = buildFullDigitalAssetURL(baseURL, filename, 'png');`.  This has me confused with different strategies across the models.

Reply:
fixed